tests/modules/encoders/test_swin_transformer_3d_encoder.py

```diff
@@ -41,41 +39,7 @@ def test_swin_transformer_3d_encoder(self):
 
         scores = self.encoder(image)
         self.assertEqual(scores.size(), torch.Size([1, 768]))
-        self.assertAlmostEqual(scores.abs().sum().item(), 277.63833, 2)
+        self.assertAlmostEqual(scores.abs().sum().item(), 247.14674, 2)
 
     def test_swin_transformer_3d_scripting(self):
         torch.jit.script(self.encoder)
-
-
-class TestSwinTransformer3dComponents(unittest.TestCase):
```

Comment:
Since we have upstreamed the model to TorchVision and import from there, there is no point in testing the components here.